rustls/src/common_state.rs

```diff
@@ -138,6 +141,18 @@ impl CommonState {
         self.suite
     }
 
+    /// Retrieves the key exchange group agreed with the peer.
+    ///
+    /// This returns None until the key exchange has been completed,
+    /// or if the [`CommonState::handshake_kind`] is
+    /// [`HandshakeKind::Resumed`].
```

Comment:
There's a bit more nuance:

- for TLS1.2, there is never a key exchange in a resumption, so this condition is accurate
- for TLS1.3, there may or may not be a key exchange in a resumption, though as it happens rustls requires this as a matter of policy